pkg/authentication/cookies.go

```diff
@@ -135,7 +148,6 @@ func (a *AuthCookieManager) clearCookie(w http.ResponseWriter, name string, path
 func (a *AuthCookieManager) getCookie(r *http.Request, name string) string {
 	cookie, err := r.Cookie(name)
 	if err != nil {
-		a.logger.Errorf("can't get cookie %s, %v", name, err)
 		return ""
```

Comment:
if we are not returning nor logging the errors do we need the if check at all? shouldn't we simply check if cookie is nil for example?

otherwise worth returning or logging the error

Reply:
The error is always "cookie not found", which we don't care, we only need the value if present
Empty string is enough to tell us no cookie was found